src/ifElse.test.js

```diff
@@ -1,11 +1,30 @@
 'use strict';
 
+const { ifElse } = require("./ifElse");
+
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('condition = true -> run first callback', () => {
+    const condition = ()=> true;
+
+    const first = jest.fn();
+    const second = jest.fn();
```

Comment:
set variables names more informative, for example firstCallback and secondCallback
fix it everywhere please

Reply:
fixed